include/client_core.h

```diff
@@ -30,9 +32,9 @@ class ClientCore {
     void render();                                  // Render the game state to the user
 
     bool connected;                // Connection state
-    Client client;                 // client.conn_sock = socket, 
+    Client client;                 // client.conn_sock = socket,
+    Packet packet;
     ServerData server_updates;
```

Comment:
Why do we have packet as a member var? Is there a single persistent packet we want the ClientCore to remember, or can this be passed between functions?

Reply:
fixed